proto/.clang-format

```diff
@@ -0,0 +1,8 @@
+---
+BasedOnStyle: Google
+IndentWidth: 4
+---
+Language: Proto
+ColumnLimit: 100
+AlignConsecutiveAssignments: true
```

Comment:
this will result in unrelated lines (from a legitimate change) getting formatted.

Reply:
yay or nay?

Reply:
yay, same as go

Reply:
_(pet peeve: the word you want in this context is "[yea](https://grammarist.com/usage/yea-yeah-yay/)")_

Reply:
(i did indeed mean yay, lol.. just a joke that will never come across on github comments)